M/MSTM/build_tarballs.jl

```diff
@@ -10,14 +10,32 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/MSTM/code
-mpifort -O2 mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 -o "${bindir}/mstm${exeext}"
+if [[ "$target" == x86_64-w64-mingw32 ]]; then
+    cd $WORKSPACE/destdir/include
+    cp $WORKSPACE/destdir/src/mpi.f90 .
+    gfortran -DWIN64 -DINT_PTR_KIND=8 -fno-range-check mpi.f90 || true
+    cd $WORKSPACE/srcdir/MSTM/code
+    echo "void __guard_check_icall_fptr(unsigned long ptr) { }" > cfg_stub.c
+    gcc -c cfg_stub.c
+    gfortran -O2 -fno-range-check mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 cfg_stub.o -L${WORKSPACE}/destdir/lib -I${WORKSPACE}/destdir/include -lmsmpifec64 -lmsmpi64 -o "${bindir}/mstm${exeext}"
+elif [[ "$target" == *-mingw* ]]; then
```

Comment:
What's the difference with the branch above?  If you want to test for 32-bit Windows (`i686-w64-mingw`) do that explicitly.

Reply:
Yes, the second branch is for 32-bit Windows. I have modified the code according to your suggestion.